### PR TITLE
[DOCFIX] Add collectInfo command to doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
This adds `collectInfo` command to `Troubleshooting.md`.

The improvements to command printouts are addressed in #11037

### CODE REVIEW

**Comment by jiacheliu3** [comment]
@ns1123 PTAL especially on the security implications. Thanks!

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -56,6 +56,69 @@ set the debug server's host and port, and start the debug session. If you set a
 will enter debug mode and you can inspect the current context's variables, call stack, thread list, and expression
 evaluation.
 
+## Alluxio collectInfo command
+
+Alluxio has a `collectInfo` command that collect information to troubleshoot an Alluxio cluster. 
+`collectInfo` will run a set of sub-commands that each collects one aspect of system information, as explained below.
+In the end the collected information will be bundled into one tarball which contains all information regarding your Alluxio cluster.
```
Comment (ns1123) [inline docs/en/operation/Troubleshooting.md]: Mention a rough estimate of size here.

File: docs/en/operation/Troubleshooting.md [hunk 2]
```diff
@@ -56,6 +56,69 @@ set the debug server's host and port, and start the debug session. If you set a
...
+
+The `collectInfo` will SSH to each node and execute the set of sub-commands.
+In the end of execution the collected information will be written to files and tarballed.
+Each individual tarball will be collected to the issuing node.
+Then all the tarballs will be bundled into the final tarball, which contains all information about the Alluxio cluster. 
+
+>NOTE: Be careful if your configuration contains credentials like AWS keys!
+You should ALWAYS CHECK what is in the tarball and REMOVE the sensitive information from the tarball before sharing it with someone!
+
+### Collect Alluxio cluster information
```
Comment (ns1123) [inline docs/en/operation/Troubleshooting.md]: Specifically mention which of these commands has credential leaking risks. That way users can know what is and isn't safe.

File: docs/en/operation/Troubleshooting.md [hunk 3]
```diff
@@ -56,6 +56,69 @@ set the debug server's host and port, and start the debug session. If you set a
...
+`collectLog` will collect all the logs under `${alluxio.work.dir}/logs`.
+
+### Collect Alluxio metrics
+`collectMetrics` will collect Alluxio metrics served at `http://${alluxio.master.hostname}:${alluxio.master.web.port}/metrics/json/` by default.
+The metrics will be collected multiple times to see the progress.
+
+### Collect JVM information
+`collectJvmInfo` will collect information about the existing JVMs on each node.
+This is done by running a `jps` command then `jstack` on each found JVM process.
+This will be done multiple times to see if the JVMs are making progress.
```
Comment (ns1123) [inline docs/en/operation/Troubleshooting.md]: Explicitly mention which nonAlluxio JVM processes we will collect information on.

File: shell/src/main/java/alluxio/cli/bundler/CollectInfo.java [hunk 4]
```diff
@@ -60,6 +61,17 @@
  */
 public class CollectInfo extends AbstractShell {
   private static final Logger LOG = LoggerFactory.getLogger(CollectInfo.class);
+  private static final String USAGE =
+          "USAGE: collectInfo [--max-threads <threadNum>] [--local]\n\n"
+                  + "collectInfo runs a set of sub-commands which collect information"
+                  + "about your Alluxio cluster. In the end of the run, "
+                  + "the collected information will be written to files "
```
Comment (ns1123) [inline shell/src/main/java/alluxio/cli/bundler/CollectInfo.java]: Usage should also reiterate credential risks. People don't read docs, but they generally take a look at the usage functionality.

**Comment by jiacheliu3** [comment]
The code change part moved to https://github.com/Alluxio/alluxio/pull/11035

File: docs/en/operation/Troubleshooting.md (hunk 3 above)
Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md]: Currently it's ALL JVMs returned by `jps` (`sudo jps` if this works). 

Are there good heuristics on deciding which JVMs we want and not want?

File: shell/src/main/java/alluxio/cli/bundler/CollectInfo.java (hunk 4 above)
Comment (jiacheliu3) [reply shell/src/main/java/alluxio/cli/bundler/CollectInfo.java]: Marked as resolved as we move the code change part to #11037

File: docs/en/operation/Troubleshooting.md (hunk 2 above)
Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md]: PTAL at the more detailed WARNINGs. Thanks!

**Comment by jiacheliu3** [comment]
Removed the change to command printouts in code. This PR focuses on the doc fix.

**Comment by ZacBlanco** [comment]
ping @ns1123

File: docs/en/operation/Troubleshooting.md (hunk 3 above)
Comment (ns1123) [reply docs/en/operation/Troubleshooting.md]: Hmm I thought we filtered for Presto/Spark. Ignore.

**Comment by jiacheliu3** [comment]
@ZacBlanco Do you mind merging this for me please? Thanks!

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**